stable/grafana/README.md

```diff
@@ -105,7 +105,7 @@ kind: ConfigMap
 metadata:
   name: sample-grafana-datasource
   labels:
-     grafana_datasource: 1
+    {{ .Release.Name }}-grafana-datasource: 1
```

Comment:
should not this use the template too?

Reply:
Well, it depends. That will be possible if this example datasource configmap (or a dashboard configmap) was part of the Grafana chart. However, it is unlikely that a setup will have such configmaps under the Grafana chart because it's best to not modify an upstream chart and just use it as-is (mostly for upgrading purposes).

These configmaps will likely be under a different chart. Perhaps the main application one, which could have Grafana chart as a dependency. I think it's best not to assume that the configmaps will be under the Grafana chart. With the current format, the defaults will work in both scenarios: configmaps outside or inside the Grafana chart.

The reason why I set it to `{{ .Release.Name }}-grafana-datasource` is just to illustrate the default value for the label.

Reply:
A better example:

If I had a `ConfigMap` in chart "x" and used `{{ include "grafana.sidecar.datasources.label" . }}`, chart "x" will fail during templating because `.Values.sidecar.datasources.label` (used in the Grafana helper partial) will likely not exist in that chart.